libraries/utils/src/shared-components/tags-input.ts

```diff
@@ -0,0 +1,52 @@
+/* eslint-disable indent */
+/* eslint-disable no-tabs */
+import * as DG from 'datagrok-api/dg';
+import * as ui from 'datagrok-api/ui';
+
+import {Observable, fromEvent} from 'rxjs';
+import {filter, map} from 'rxjs/operators';
+
+import '../../css/tags-input.css';
+
+
+export class TagsInput extends DG.InputBase {
+  tags: string[];
+  _tagsDiv: HTMLDivElement;
+
+  constructor(tags: string[]) {
+    const inputElement = ui.stringInput('', '');
+    super(inputElement.dart);
+
+    this.tags = tags;
+    this._tagsDiv = ui.div(tags.map((tag) => ui.span([ui.span([tag])], `ui-tag ${tag}`)), 'ui-tag-list');
+    this.root.append(this._tagsDiv);
+
+    this._initEventListeners();
+  }
+
+	_initEventListeners() {
+		this.input.addEventListener('keyup', (event: KeyboardEvent) => {
+      if (event.code === 'Enter')
+        this.addTag((this.input as HTMLInputElement).value);
+    });
+	}
```

Comment:
Something is wrong with the formatting of the function `_initEventListeners`

Reply:
I still can't understand why it's displayed like that, because I don't have wrong tabulation locally and it is pushed